source/configuration_manual/zlib_plugin.rst

```diff
@@ -34,11 +34,14 @@ supported values are: gz, bz2, lz4, zstd) to use when saving a new mail.
 The ``zlib_save_level`` setting sets the compression level used.
 
 .. note::
-   Currently, the compression level must be an integer in the range 1 to 9
+   Prior to 2.3.15+, the compression level must be an integer in the range 1 to 9
    regardless of the algorithm selected.  The default level is 6.  These
    values may not be sensical with compression algorithms other than gz and
    bz2.
 
+   Since 2.3.15+ it accepts the range accepted by the algorithm, please refer
+   to documentation of each algorithm to figure out the accepted range.
```

Comment:
.. versionchanged:: v2.3.15 ?

Reply:
s/figure out/find out/ sounds a bit nicer

Reply:
better

Reply:
fs-compress and zlib plugin pages have now different text explaining the same change. Since the code is the same, shouldn't the documentation be identical as well?

Reply:
now?